Kiss.Bff/Dockerfile

```diff
@@ -51,6 +51,7 @@ RUN dotnet publish "Kiss.Bff.csproj" -c Release -o /app/publish /p:UseAppHost=fa
 FROM base AS final
 WORKDIR /app
 COPY certificates /usr/local/share/ca-certificates
+COPY ./Kiss.Bff/json .
```

Comment:
is dit nodig? de json files worden toch gewoon meegepublished met .net project?
Anyway, volgens mij zijn losse json files hier niet de beste oplossing.
Liever in de seedcontroller de items instantiëren. het voordeel is dat je dan meteen compiletime zeker weet dat ze valide zijn. Als je bestanden uitleest weet je het pas runtime. Als iemand bijvoorbeeld het model in de toekomst wijzigt en niet weet dat die losse json files er zijn, dan merk je pas in gebruik dat er iets stuk is gegaan.

dus ipv `var berichten = ReadListFromJsonFile<Bericht>(_environment, "berichten.json");` kan je zoiets doen:
```
 var berichten = new List<Bericht> { 
    new(){
        DateCreated = DateTime.Now,
        PublicatieDatum = DateTime.Now,
        Titel="Dit is een voorbeeld Werkinstructie",
        Inhoud="<p>Voorbeeld tekst</p>",
        Type="Werkinstructie"
    },
    new(){
        DateCreated = DateTime.Now,
        PublicatieDatum = DateTime.Now,
        Titel="Dit is een voorbeeld Nieuwsbericht",
        Inhoud="<p>Voorbeeld tekst</p>",
        Type="Nieuws"
    }
};
```

Reply:
aanvulling: als we het idd niet met json files gaan doen, dan graag wel de content uit (zojuist aangepaste) jsonfiles gebruiken. Omdat ik daarin specifieke nieuwsberichten en werkinstructies heb opgenomen, met informatie over KISS, over beheer van nieuws en werkinstructies en links naar de documentatie. Ook bij de Links heb ik verwijzingen opgenomen naar de documentatie.

Reply:
> aanvulling: als we het idd niet met json files gaan doen, dan graag wel de content uit (zojuist aangepaste) jsonfiles gebruiken. Omdat ik daarin specifieke nieuwsberichten en werkinstructies heb opgenomen, met informatie over KISS, over beheer van nieuws en werkinstructies en links naar de documentatie. Ook bij de Links heb ik verwijzingen opgenomen naar de documentatie.

Ja, aangepaste json gezien, die zal ik gebruiken.

Reply:
Aangepast: item instances op basis van de aangepaste json